package.json

```
@@ -71,8 +71,11 @@
     "start": "react-scripts start",
     "init": "node scripts/init.js",
     "build": "react-scripts build",
-    "lint": "prettier --write \"{src,scripts,functions,configs,reporting}/**/*.{js,jsx,ts,tsx,json,css,scss,md}\"",
-    "lint:src": "eslint src --ext .js,.jsx,.ts,.tsx",
+    "lint": "npm run prettier:check && npm run eslint:src",
+    "eslint:src": "eslint src --ext .js,.jsx,.ts,.tsx",
+    "eslint:fix": "eslint src --fix --ext .js,.jsx,.ts,.tsx",
```

[OOS, Future Cleanup] Can potentially rewrite this as:
```suggestion
    "eslint:fix": "npm run eslint:src --fix,
```